packages/openapi-generator/src/parser/operation.ts

```diff
@@ -72,13 +72,33 @@ function isPlaceholder(pathPart: string): boolean {
   return /^\{.+\}$/.test(pathPart);
 }
 
+function validatePlaceholder(pathPart: string): boolean {
+  return /^\{[^{}]+\}$/.test(pathPart);
+}
+
+function validatePathPattern(
```

Comment:
General: The methods are called validateXYZ and give back a boolean. For boolean I like the `isSomeThing` form. For validate I expect that the method throws an exeception if not valid.

Reply:
Also here add some small comment what the method does.